### ISSUE TITLE
Property Changes on Continental Shelves

### ISSUE THREAD

**Issue by kathygunn** [adele157/IAF-meltwater#4]
How do the properties change on the shelf, and how does that compare to observations?

Bottom temperature, bottom salinity, stratification, oxygen solubility

**Comment by matthew-england-unsw** [comment]
Would be good to evaluate the T-S tendencies to compare with this plot from observations:
[Jacobs_et_al_plot.pdf](https://github.com/adele157/IAF-meltwater/files/9465909/Jacobs_et_al_plot.pdf)

**Comment by matthew-england-unsw** [comment]
Would be good to evaluate the T-S tendencies to compare with this plot from observations:

[Jacobs_et_al_plot.pdf](https://github.com/adele157/IAF-meltwater/files/9465916/Jacobs_et_al_plot.pdf)

**Comment by kathygunn** [comment]
I have focussed on salinity changes.

The figure shows spatial- and depth-mean practical salinities over some windowed region and depth range following Jacobs et al (2022) for Terra Nova Bay. They compare the control, MW peturb, and observational data. 


<img width="891" alt="shelf_salins_TNB" src="https://user-images.githubusercontent.com/51481219/189040987-6c8ae67c-6429-4240-bf70-6527cd0a91eb.png">

**Comment by kathygunn** [comment]
Moving east to Ross Island
<img width="859" alt="shelf_salins_Ross_Island" src="https://user-images.githubusercontent.com/51481219/189041052-95177cf4-f140-4c9a-8465-39343b294ce9.png">

**Comment by kathygunn** [comment]
Moving further east to Little America Bay
<img width="877" alt="shelf_salins_Little_America_Bay_" src="https://user-images.githubusercontent.com/51481219/189041115-49163f6c-e006-451f-ab12-a17a446dea54.png">

**Comment by kathygunn** [comment]
Thoughts so far:

1) The difference in the temporal behaviour between the mean salinity in the control and meltwater perturbations are not as large as I was expecting. 
2) In the meltwater perturbations, the temporal salinity changes on the shelf are not what I was expecting either; there appears to be no long-term trend, rather abrupt shifts with time. 

Keen to hear what others think...

**Comment by adele157** [comment]
Just changing this issue title to focus on the Ross Sea, because it will get pretty messy if we include other regions here too - let's start new issues for other location obs comparisons.

**Comment by adele157** [comment]
This is really interesting, thanks @kathygunn! It looks like the wind/atmospheric variability is a stronger driver than than the meltwater forcing. i.e. those abrupt shifts in time match between the control and meltwater simulations, so are likely wind-driven. 

Since most of those large jumps aren't in the observations, I guess that means that the model overestimates the magnitude of the wind-driven variability here.

Also cool to see that the model gets the recent Terra Nova Bay salinity recovery - we had looked for this before in the model, but I guess we were averaging over larger domains where it's not so well simulated. The increase in the 80s and drop in 1990 also look similar in TNB (but not 1995-2010...).

Does this mean that we've underestimated the magnitude of the meltwater perturbation? Maybe the [salinity restoring](https://github.com/adele157/IAF-meltwater/issues/2) is too strong? Can we check this in other locations, like Amundsen trends perhaps? Or abyssal trends?

**Comment by adele157** [comment]
Also note that the control extends back to 1958 - it would be worth extending those lines back in time to see what happens in the control earlier on.

**Comment by kathygunn** [comment]
@adele157 - yes it certainly looks like the wind/atmospheric variability is stronger than the meltwater forcing. Yes, that could be the case in terms of the model over-estimating the response to wind... it may also be that the obs are smoothing those changes over. 
Given these points, it also makes sense that the TNB salinity recovery is captured, as that was ultimately wind-driven changes in sea ice.

I am going to make an issue for the Amundsen Sea to do a similar analysis.

**Comment by kathygunn** [comment]
Compare with Guo et al (2020) Fig. 2a https://iopscience.iop.org/article/10.1088/1748-9326/abc995/meta which does not seem to show a trend

**Comment by adele157** [comment]
Just checking, @kathygunn, are these annual averages or summer averages from the model? The Jacobs et al. 2022 paper uses only summer obs right? Possibly there's not much difference at 500m depth, I'm not sure.

Also, when you say, the Ross Sea salinity rebound "was ultimately wind-driven changes in sea ice". Is that just referring to the Silvano et al 2020 paper? I thought they proposed a nice mechanism that sounds plausible, but I was 100% convinced by their evidence for it. Do we know if anyone else has looked at the mechanisms behind the Ross Sea rebound?

**Comment by kathygunn** [comment]
@adele157: yes, the Jacobs et al (2022) data are summer means (green line). I've now added some annual averages from Terra Nova Bay to the relevant plot. These new data (red line) are from Guo et al (2020) who take means below 600 m in TNB using WOD measurements. Note that, I have not take annual means of the model output, but eye-balling it you can see that the model and obs will not match up.

It seems that the model (with and without freshwater forcing) is not capturing the long-term freshening trend in this location.

![Screen Shot 2022-09-14 at 13 53 40](https://user-images.githubusercontent.com/51481219/190055826-1afbd535-48f8-4b35-a927-9f6601d82295.png)


Yes, that was just referencing the Silvano et al (2020) paper. Interesting to hear your thoughts, and keen to chat about it more.

**Comment by kathygunn** [comment]
Jacobs et al (2022) Fig 1 showing where Ross Sea regions are:
![Screen Shot 2022-09-15 at 11 29 42](https://user-images.githubusercontent.com/51481219/190292177-cea63395-f727-4537-8ffd-9fe2675256ba.png)

**Comment by kathygunn** [comment]
TNB (with DJF model means, control extended back to 1958, and modelled temperature included)
<img width="460" alt="TNB_Bathym" src="https://user-images.githubusercontent.com/51481219/190292311-5d0f0710-642e-4a4e-89f1-72df74146993.png">
<img width="890" alt="DJF_TNB" src="https://user-images.githubusercontent.com/51481219/190292321-695b37ac-01a7-46d2-b32d-4d868eaaa92b.png">

**Comment by kathygunn** [comment]
Ross Island (with DJF model means, control extended back to 1958, and modelled temperature included)
<img width="469" alt="Ross_Island_Bathym" src="https://user-images.githubusercontent.com/51481219/190292379-1707a627-dfe2-4f5a-9195-a712327f2579.png">
<img width="874" alt="Screen Shot 2022-09-15 at 11 33 47" src="https://user-images.githubusercontent.com/51481219/190292590-31816dd4-730b-47ef-b6c6-6a0bdf5e0c88.png">
